external/common/lapack/FindMathOpenMP.cmake

```diff
@@ -73,44 +73,72 @@ endmacro()
 
 add_library(tgt::MathOpenMP INTERFACE IMPORTED)
 
-if (${isMKL} MATCHES "MKL")
-    if ((CMAKE_C_COMPILER_ID STREQUAL GNU) OR
-        (CMAKE_CXX_COMPILER_ID STREQUAL GNU) OR
-        (CMAKE_Fortran_COMPILER_ID STREQUAL GNU))
-        if (APPLE)
-            set(_MathOpenMP_LIB_NAMES "iomp5")
+if(WIN32)
```

Comment:
This conditional stanza should go into `FindTargetOpenMP.cmake` which detects OpenMP functionality _apart from_ math libraries and returns the `OpenMP::OpenMP` target (with compiler flags, definitions, include directories, and link libraries). Specifically, I think [these lines](https://github.com/psi4/psi4/blob/master/external/common/lapack/FindTargetOpenMP.cmake#L71-L91) should be extended

Reply:
That part seems to be broken, as the compiler flags are not passed.

I'll try to Windows configuration, but some hacks, which I don't fully understand might stop working!